README.md

```diff
@@ -4,7 +4,69 @@ bookend is a classifier for text data. We are building a machine learning model
+
+will it know Charles Dickens wrote it?
+
+### Methodology
+
+We build many features out of the text, and use an ensemble machine learning model to predict the author.
+
+In the future we will expand to allow any-size snippets of text and additional authors, but for now we have focused on identifying 75-sentence chunks of text. We include the following authors, chosen as the top authors on Project Gutenberg who wrote their works in English (We exclude Shakespeare, as his works are largely plays rather than novels or short stories. We additionally add JK Rowling, who famously wrote the Harry Potter series as well as crime fiction novels under the pseudonym Robert Galbraith):
+
+| Author                | Jane Austen | Lewis Caroll | Daniel Defoe | Charles Dickens | Sir Arthur Conan Doyle | Jack London | J.K. Rowling | Mary Shelley | Robert Louis Stevenson | H.G. Wells | Oscar Wilde |
+|-----------------------|-------------|--------------|--------------|-----------------|------------------------|-------------|--------------|--------------|------------------------|------------|-------------|
+| Works in Training Set |             |              |              |                 |                        |             |              |              |                        |            |             |
+| Work(s) Testing Set   |             |              |              |                 |                        |             |              |              |                        |            |             |
+|                       |             |              |              |                 |                        |             |              |              |                        |            |             |
+
+Below we briefly summarize several avenues we investigated for engineering features from the text. For more detail, we refer you to the [notebooks](https://github.com/data-dart/bookend/tree/master/notebooks) and [source code](https://github.com/data-dart/bookend/tree/master/src).
+
+#### Bag-of-words
+
+We build a "vocabulary" of known words, assembled from the unique [lemmatized](https://en.wikipedia.org/wiki/Lemmatisation) words in the corpus. For each text sample, we build a feature vector indicating how many times each vocabulary word appeared.
+
+#### Lexical Features
+
+These features are a grab-bag of word and sentence level statistics, like the mean and variation in word and sentence length, and the [Flesch-Kincaid grade level](https://en.wikipedia.org/wiki/Flesch%E2%80%93Kincaid_readability_tests#Flesch%E2%80%93Kincaid_grade_level).
+
+#### Syntactic Features
+
+These features focus on the frequency of different parts of speech used by the authors.
+
+#### N-Grams
+
+These features look at [n-grams](https://en.wikipedia.org/wiki/N-gram), or the frequencies with which n words appear together contiguously (for example, the phrase "how are you today?" has 2-grams "how are," "are you," and "you today.").
```

Comment:
Please replace with the following:

These features look at the relationship between sets of words and/or characters in a text. For example, relatively how often does the bi-gram (2 word sequence) "how are" appear between "hello how" and "are you"? We can compute the relative similarity between texts in this way by creating graphs of each text, where the nodes contain the n-grams and the edges contain the number of times each node is adjacent to another node, and comparing them using standard graph theory.

Reply:
Done.